webview/src/experiments/components/table/TableHeader.tsx

```diff
@@ -271,6 +286,15 @@ const TableHeaderCell: React.FC<{
         key={column.id}
         role="columnheader"
         tabIndex={0}
+        onMouseEnter={() => {
```

Comment:
Do we need to write some tests for these changes? I wasn't sure 😅

Reply:
Yes please!